docs/syntax_and_semantics/assignment.md

```diff
@@ -58,6 +61,9 @@ list.[]=(2, 3)
 list[2] = 3
 ```
 
+NOTE: This does not apply to top-level methods. An assignment `x = y` in the top-level scope is always an assignment to
```

Comment:
NOTE: This will just be the literal text `NOTE:`

Unless you're ready to enable https://oprypin.github.io/markdown-callouts/ :D